### ISSUE TITLE
Custom theme properties not detected from tailwindcss 4

### ISSUE THREAD

**Issue by GALWR** [schoero/eslint-plugin-readable-tailwind#296]
### Description

When custom properties like colors are defined in the theme section, tailwindcss uses them to create additional class names, which work in the layout but trigger the rule `better-tailwindcss/no-unknown-classes`.

For example:

FIle: index.css
```css
@import "tailwindcss";

:root {
  --background: #ffffff;
  --foreground: #171717;
}

@theme inline {
  --color-background: var(--background);
  --color-foreground: var(--foreground);
  --color-primary: #e60012;
  --color-secondary: #9fa0a0;
  --color-accent: var(--color-primary);
  --color-neutral: var(--foreground);
  --color-light: #f3f3f3;
  --color-disabled: var(--color-secondary);
}
```

```layout.tsx
import type { FC, ReactNode } from 'react';

const MainLayout: FC<Readonly<{
  children: ReactNode;
}>> = ({ children }) => (
  <>
    <header className="bg-primary text-white">
      Main Header
    </header>

    <main className="grow">
      {children}
    </main>

    <footer className="bg-primary h-8 text-white">
      Footer
    </footer>
  </>
);

export default MainLayout;

```

The classnames `bg-primary` and `bg-secondary` trigger the rule

### Screenshot

<img width="966" height="500" alt="Image" src="https://github.com/user-attachments/assets/6deee9fd-c73c-415f-8ae6-045615c13f36" />

.

### Request

Please add support for custom property detection

### Tailwindcss feature docs

https://tailwindcss.com/docs/adding-custom-styles#customizing-your-theme

**Comment by schoero** [comment]
This is already supported.
Make sure that you correctly set up the `entryPoint`: https://github.com/schoero/eslint-plugin-better-tailwindcss/blob/main/docs/parsers/tsx.md
